pkg/controller/dynconfig/dynconfig.go

```diff
@@ -197,6 +197,14 @@ func (d *DynConfig) dynamicUpdateBackends() bool {
 		curBackend := d.curBackendsMap[backendName]
 		updBackend := d.updBackendsMap[backendName]
 
+		if updBackend.DNSResolvers.UseResolver != "" {
+			if updBackend.DNSResolvers.UseResolver != curBackend.DNSResolvers.UseResolver {
+				glog.Infof("DNS used for %s\n", backendName)
+				reloadRequired = true
+			}
+			continue
+		}
```

Comment:
* Say a few words why to test `UseResolver != ""`
* This `continue` above should only happen if the only difference between old and cur is the endpoints, so move the whole if statement after the `if !reloadRequired {}` (below)

Reply:
in previous commits where use_resolver was used differently that was required, 
now all when it only used for logging some info i can clean it.